src/components/Match/matchColumns.jsx

```diff
@@ -438,18 +431,18 @@ export const supportColumns = [
 export const chatColumns = [
   {
     displayName: strings.filter_is_radiant,
-    field: 'team',
-    displayFn: row =>
+    displayFn: row => 
```

Comment:
Sure. What exactly is the field value's intended use? I assumed it was related to sorting, and excluding it worked fine on my local.